sections/faqs/extend-and-styled-difference.js

```diff
@@ -0,0 +1,25 @@
+import md from 'components/md'
+
+const ExtendAndStyled = () => md`
+  ## What is the difference between \`Comp.extend\` and \`styled(Comp)\`?
+
+  ### Behind the scenes
+  The \`styled()\` factory generates new component styles with a new class. The classnames are
+  then passed to the React component via the \`className\` prop.
+  Calling \`extend\` creates new component styles by extending the old one, and thus doesn't
+  generate two classes for a single component. (\`styled()\` factory does that)
+
+  ### Which one should you use
+  We recommend to use \`.extend\` where possible for internal \`StyledComponents\`,
+  but the \`styled()\` factory for external components and cross-file containers/components
+  where it isn't clear what it's going to be when writing the wrapper.
+
+  > Wrapping a container is not recommended. It should always be attempted to wrap
```

Comment:
Let's not make this a note but integrate this into the normal text as it's part of the explanation.

```diff
- Wrapping a container is not recommended.
+ Keep in mind that wrapping your own container and structural components is not recommended.
```